code/modules/mob/living/basic/space_fauna/giant_spider/spider_abilities/hivemind.dm

```diff
@@ -0,0 +1,70 @@
+/**
+ * Sets a directive to be given to all future spiders created by the user.
+ * This will be overwritten if used again.
+ */
+/datum/action/set_spider_directive
+	name = "Set Directive"
+	desc = "Set a directive for your future children to follow."
+	button_icon = 'icons/mob/actions/actions_animal.dmi'
+	button_icon_state = "directive"
+	background_icon_state = "bg_alien"
+	overlay_icon_state = "bg_alien_border"
+	check_flags = AB_CHECK_CONSCIOUS
+	/// Current directive to apply
+	var/current_directive
```

Comment:
```suggestion
	/// Current directive to apply
	var/current_directive = ""
```

Type it to a string since it is one, do this in conjuction with the one where we don't mutate current_directive in the action's Trigger proc.